src/ontology/components/mondo-tags.owl

```diff
@@ -25443,9 +25443,7 @@
 
     <!-- http://purl.obolibrary.org/obo/MONDO_0016676 -->
 
-    <owl:Class rdf:about="http://purl.obolibrary.org/obo/MONDO_0016676">
-        <mondo:excluded_from_qc_check rdf:resource="http://purl.obolibrary.org/obo/mondo/sparql/qc/general/qc-single-child.sparql"/>
-    </owl:Class>
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/MONDO_0016676"/>
```

Comment:
are the changes in this file intended?

Reply:
yes, it is an automatic change because I removed the "mondo:excluded_from_qc_check" annotation